cross/p7zip/Makefile

```diff
@@ -13,13 +13,10 @@ COMMENT  = 7-Zip is a file archiver with a high compression ratio.
 LICENSE = LGPLv2.1+
 
 CONFIGURE_TARGET = nop
-INSTALL_TARGET = p7zip_install
 
 COMPILE_MAKE_OPTIONS += CC="$(TC_PATH)$(TC_PREFIX)gcc"
 COMPILE_MAKE_OPTIONS += CXX="$(TC_PATH)$(TC_PREFIX)g++"
 
-include ../../mk/spksrc.cross-cc.mk
+INSTALL_MAKE_OPTIONS = install DEST_DIR=$(INSTALL_DIR) DEST_HOME=$(INSTALL_PREFIX)
```

Comment:
I wonder if `GNU_CONFIGURE` could have (or not) an effect on how we setup `INSTALL_MAKE_OPTIONS` such as below (or the reverse).  I haven't further looked if it made any sense, just sharing a thought.
```
ifeq ($(strip $(INSTALL_MAKE_OPTIONS)),)
ifef ($(strip $(GNU_CONFIGURE)),1)
INSTALL_MAKE_OPTIONS = install DESTDIR=$(INSTALL_DIR) prefix=$(INSTALL_PREFIX)
else
INSTALL_MAKE_OPTIONS = install DEST_DIR=$(INSTALL_DIR) DEST_HOME=$(INSTALL_PREFIX)
endif
endif
```